modules/understanding-upgrade-channels.adoc

```diff
@@ -7,79 +7,89 @@
 // * updating/updating-disconnected-cluster.adoc
 
 [id="understanding-upgrade-channels_{context}"]
-= Understanding {product-title} upgrade channels
+= {product-title} Upgrade Channels and Update Types
```

Comment:
I like this, because all of our docs should be about understanding.  But the file name also has `understanding-`.  Can we fix that too?  @kalexand-rh, is there history/policy we're missing?